src/Tracy/Bar.php

```diff
@@ -222,4 +214,34 @@ public function dispatchContent()
 		}
 	}
 
+	private function getUrlPath()
+	{
+		$path = isset($_SERVER['REQUEST_URI']) ? explode('?', $_SERVER['REQUEST_URI'])[0] : '/';
+		$lpath = strtolower($path);
+		$script = isset($_SERVER['SCRIPT_NAME']) ? strtolower($_SERVER['SCRIPT_NAME']) : '';
+		if ($lpath !== $script) {
+			$max = min(strlen($lpath), strlen($script));
+			for ($i = 0; $i < $max && $lpath[$i] === $script[$i]; $i++) {
+				;
+			}
+			$path = $i ? substr($path, 0, strrpos($path, '/', $i - strlen($path) - 1) + 1) : '/';
+			$cookiePath = session_get_cookie_params()['path'];
+			if (substr($cookiePath, 0, strlen($path)) === $path) {
+				$path = rtrim($cookiePath, '/') . '/';
+			}
+		}
+		return $path;
+	}
+
+	private function getStopXdebug()
+	{
+		return extension_loaded('xdebug')
+			? ['XDEBUG_SESSION_STOP' => 1, 'XDEBUG_PROFILE' => 0, 'XDEBUG_TRACE' => 0] : [];
+	}
+
+	private function getTracyDebugScriptPath($path)
+	{
+		return htmlspecialchars("$path?")
+		. http_build_query(['_tracy_bar' => 'js', 'v' => Debugger::VERSION] + $this->getStopXdebug(), NULL, '&amp;');
+	}
```

Comment:
I have no idea what have you changed because you combined refactoring and maybe some hidden change into single commit.